code/source/playfab/QoS/PlayFabQoSApi.cpp

```diff
@@ -234,7 +234,8 @@ namespace PlayFab
             auto a = result.QosServers;
             for (auto it = a.begin(); it != a.end(); ++it)
             {
-                api->regionMap[it->Region] = move(it->ServerUrl);
+                //api->regionMap[it->Region] = move(it->ServerUrl);
+                api->regionMap["it->Region as a std::string"] = move(it->ServerUrl);
```

Comment:
This looks like the singular string "it->Region as a std::string" will be the only key in our regionMap? Should there be a cast or helper conversion function?

Reply:
it-Region will become a string when models are generated using your new API specs

Reply:
In other words, the commented line should be the real code, not the line that I included in this PR